catalogue/aarch64-MTE/shelf.py

```diff
@@ -1150,4 +1150,61 @@
     "tests/SB+postp-addrs.litmus",
     "tests/SB+rfitt-addrtps.litmus",
     "tests/SB+rfitt-dmb.ldtps.litmus",
+    "tests/from-tagcheck-async/LB-notagfault+po+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+addr+dmb.ldpt--async.litmus",
+    # "tests/from-tagcheck-async/LB-tagfault+ctrl+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+data+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+dmb.ish+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+dmb.ld+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+dsb.ld+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+po+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/LB-tagfault+rel+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-async/MP+dmb.stpt+tcib-po-isb-po--async.litmus",
+    "tests/from-tagcheck-async/MP+dmb.stpt+tcib-tagfault-po--async.litmus",
+    "tests/from-tagcheck-async/S+dmb.stpt+tcib-po--async.litmus",
+    "tests/from-tagcheck-async/S-tagfault+dmb.st+dmb.ldpt--async.litmus",
+    "tests/from-tagcheck-sync/LB-notagfault+po+dmb.ldpt--sync.litmus",
+    "tests/from-tagcheck-sync/LB-tagfault+po+dmb.ldpt--sync.litmus",
+    "tests/from-tagcheck-sync/MP+dmb.stpt+tcib-po-isb-po.litmus",
+    "tests/from-tagcheck-sync/MP+dmb.stpt+tcib-tagfault-po.litmus",
+    "tests/from-tagcheck-sync/S+dmb.stpt+tcib-po.litmus",
```

Comment:
These 3 tests are missing from the repository, I think you might have missed adding them to the relevant commit.

Reply:
Oops! Sorry about this, should be fixed now.